diff -ur otcl-1.8.orig/conf/configure.in.tcl otcl-1.8/conf/configure.in.tcl
--- otcl-1.8.orig/conf/configure.in.tcl	2003-02-10 23:44:15.000000000 +0100
+++ otcl-1.8/conf/configure.in.tcl	2004-08-18 20:51:38.129842400 +0200
@@ -130,6 +130,7 @@
 dnl /usr/include/tcl is for Debian Linux
 TCL_LIB_PLACES_D="$d \
 		$d/lib \
+		$d/lib64 \
 		$d/unix"
 TCL_LIB_PLACES=" \
 		../lib \
@@ -156,9 +157,11 @@
 		/usr/lib/tcl$TCL_OLD_ALT_VERS \
                 /usr/local/lib \
                 $prefix/lib \
+                $prefix/lib64 \
                 $x_libs/tk \
                 $x_libs \
                 /usr/lib \
+                /usr/lib64 \
 		"
 
 dnl Decide which set of .tcl library files to use
diff -ur otcl-1.8.orig/conf/configure.in.tk otcl-1.8/conf/configure.in.tk
--- otcl-1.8.orig/conf/configure.in.tk	2003-02-10 23:44:15.000000000 +0100
+++ otcl-1.8/conf/configure.in.tk	2004-08-18 20:52:07.275411600 +0200
@@ -118,6 +118,7 @@
 dnl /usr/include/tcl for debian/linux
 TK_LIB_PLACES_D="$d \ 
                 $d/lib \
+                $d/lib64 \
 		$d/unix"
 TK_LIB_PLACES=" \
 		../lib \
@@ -142,9 +143,11 @@
                 /usr/local/lib/tk$TK_OLD_ALT_VERS \
                 /usr/local/lib \
                 $prefix/lib \
+                $prefix/lib64 \
                 $x_libs/tk \
                 $x_libs \
-                /usr/lib"
+                /usr/lib \
+                /usr/lib64"
 
 NS_BEGIN_PACKAGE(tk)
 NS_CHECK_HEADER_PATH(tk.h,$TK_H_PLACES,$d,$TK_H_PLACES_D,V_INCLUDE_TK,tk)
diff -ur otcl-1.8.orig/conf/configure.in.x11 otcl-1.8/conf/configure.in.x11
--- otcl-1.8.orig/conf/configure.in.x11	2003-02-10 23:44:15.000000000 +0100
+++ otcl-1.8/conf/configure.in.x11	2004-08-18 20:52:20.640379816 +0200
@@ -4,6 +4,7 @@
 xlibdirs="\
 	/usr/openwin/lib \
 	/usr/X11R6/lib \
+	/usr/X11R6/lib64 \
 	/usr/lib/X11R6 \
 	/usr/X11R5/lib \
 	/usr/lib/X11R5 \
